source/api/content-state/0.3/index.md

```diff
@@ -0,0 +1,675 @@
+
+
+### 4.3. Multiple targets for a comparison view
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/{{ page.major }}/context.json",
+  "id": "https://example.org/import/3",
+  "type": "Annotation",
+  "motivation": "contentState",
+  "target": [
+      {
+          "id": "https://example.org/iiif/item1/canvas37",
+          "type": "Canvas",
+          "partOf": [
+              {
+                  "id": "https://example.org/iiif/item1/manifest",
+                  "type": "Manifest"
+              }
+          ]
+      },
+      {
+          "id": "https://example.org/iiif/item2/canvas99",
+          "type": "Canvas",
+          "partOf": [
+              {
+                  "id": "https://example.org/iiif/item2/manifest",
+                  "type": "Manifest"
+              }
+          ]
+      }
+  ]
+}
+```
+
+Here the viewer should open two manifests at once (if it is capable of such a view).
+
+
+### 4.4. Search Results
+
+The following example uses the compact, query string form of the content state to demonstrate what HTML search results linking to a particular viewer might look like.
+
+Firstly, in non-valid, unencoded form to show the annotation:
+
+```html
+<h2>Results for "cats"</h2>
+<ol>
+  <li>
+    <h3><a href='viewer.html?iiif-content={"id":"https://example.org/alice/canvas77#xywh=1000,2000,1000,2000","type":"Canvas","partOf":[{"id":"https://example.org/alice/manifest","type":"Manifest"}]}'>Alice in Wonderland</a></h3>
+    <p>...she has often seen a <b>cat</b> without a grin but never a grin without a <b>cat</b></p>
+  </li>
+  <!-- ... more results -->
+</ol>
+```
+
+...and then in encoded form:
+
+```html
+<h2>Results for "cats"</h2>
+<ol>
+  <li>
+    <h3><a href='viewer.html?iiif-content=eyJpZCI6Imh0dHBzOi8vZXhhbXBsZS5vcmcvYWxpY2UvY2FudmFzNzcjeHl3aD0xMDAwLDIwMDAsMTAwMCwyMDAwIiwidHlwZSI6IkNhbnZhcyIsInBhcnRPZiI6W3siaWQiOiJodHRwczovL2V4YW1wbGUub3JnL2FsaWNlL21hbmlmZXN0IiwidHlwZSI6Ik1hbmlmZXN0In1dfQ=='>Alice in Wonderland</a></h3>
+    <p>...she has often seen a <b>cat</b> without a grin but never a grin without a <b>cat</b></p>
+  </li>
+  <!-- ... more results -->
+</ol>
+```
+
+
+## Appendices
+
+### A. Acknowledgements
+{: #acknowledgements}
+
+Many thanks to the members of the [IIIF community][iiif-community] for their continuous engagement, innovative ideas, and feedback.
+
+Many of the changes in this version are due to the work of the [IIIF Discovery Technical Specification Group][groups-discovery], chaired by Antoine Isaac (Europeana), Matthew McGrattan (Digirati) and Rob Sanderson (J. Paul Getty Trust). The IIIF Community thanks them for their leadership, and the members of the group for their tireless work.
+
+### B. Change Log
+{: #change-log}
+
+| Date       | Description           |
+| ---------- | --------------------- |
+| 2018-10-31 | Version 0.1 (unnamed) |
```

Comment:
Do we need to add version 0.3 to this?

Reply:
yes

Reply:
And @julsraemy points out that it is our practice to have latest version first